

while True:
    try:
        print(")")
    except ValueError:
        print("ERROR")
        exit()